packages/cosmos/Cargo.toml

```diff
@@ -25,7 +25,7 @@ serde_json = "1"
 chrono = "0.4.31"
 hex = "0.4"
 sha2 = "0.10"
-reqwest = { version = "0.11.14", default-features = false, features = ["json", "gzip", "brotli"] }
+reqwest = { version = "0.11.14", default-features = false, features = ["json", "gzip", "brotli", "rustls-tls"] }
```

Comment:
Seems like there's multiple options for a tls implementation. I'm not sure this is the best one, but I know I've used it in the past and it worked.

https://docs.rs/crate/reqwest/latest/features